src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,14 +1,34 @@
 import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { changeQuary, changeStatus } from '../../features/filter';
+import { Status } from '../../types/Status';
+import { RootState } from '../../app/store';
 
 export const TodoFilter: React.FC = () => {
+  const { status, query } = useSelector((state: RootState) => state.filter);
+
+  const dispatch = useDispatch();
+
+  const handleSelectFilter = (statusPayload: Status) => {
+    dispatch(changeStatus(statusPayload));
+  };
+
+  const handleChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    handleSelectFilter(event.target.value as Status);
+  };
+
+  const handleQuery = (queryPayload: string) => {
+    dispatch(changeQuary(queryPayload));
+  };
+
   return (
     <form
       className="field has-addons"
       onSubmit={event => event.preventDefault()}
     >
       <p className="control">
         <span className="select">
-          <select data-cy="statusSelect">
+          <select data-cy="statusSelect" value={status} onChange={handleChange}>
```

Comment:
if u have 3 or more attributes, move them to separate lines
```suggestion
          <select 
            data-cy="statusSelect" 
            value={status} 
            onChange={handleChange}
          >
```

Reply:
I can't do this because if I display the information in three rows, after saving, it automatically returns to the blackberry row